packages/devtools/src/commands/elementClick.js

```diff
@@ -0,0 +1,22 @@
+export default async function elementClick ({ elementId }) {
+    const page = this.windows.get(this.currentWindowHandle)
+    const elementHandle = this.elementStore.get(elementId)
+
+    if (!elementHandle) {
+        throw new Error(`Couldn't find element with id ${elementId} in cache`)
+    }
+
+    /**
+     * ensure to fulfill the click promise if the click has triggered an alert
+     */
+    await new Promise((resolve) => {
```

Comment:
if element is no longer visible or something is wrong I'm getting
```
[0-0] (node:10140) UnhandledPromiseRejectionWarning: Error: Node is either not visible or not an HTMLElement
    at ElementHandle._clickablePoint (/Users/mygr/sources/e2e/node_modules/puppeteer-core/lib/JSHandle.js:199:13)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  -- ASYNC --
    at ElementHandle.<anonymous> (/Users/mygr/sources/e2e/node_modules/puppeteer-core/lib/helper.js:111:15)
    at resolve (/Users/mygr/sources/e2e/node_modules/devtools/build/commands/elementClick.js:26:26)
    at new Promise (<anonymous>)
    at DevToolsDriver.elementClick (/Users/mygr/sources/e2e/node_modules/devtools/build/commands/elementClick.js:22:9)
    at Element.<anonymous> (/Users/mygr/sources/e2e/node_modules/devtools/build/devtoolsdriver.js:88:47)
    at Element.runCommand (/Users/mygr/sources/e2e/node_modules/@wdio/sync/build/wrapCommand.js:101:21)
    at Element.wrapCommandFn (/Users/mygr/sources/e2e/node_modules/@wdio/sync/build/wrapCommand.js:35:25)
    at Element.runCommand (/Users/mygr/sources/e2e/node_modules/@wdio/sync/build/wrapCommand.js:101:21)
    at Element.wrapCommandFn (/Users/mygr/sources/e2e/node_modules/@wdio/sync/build/wrapCommand.js:35:25)
    at Element.elementErrorHandlerCallbackFn (/Users/mygr/sources/e2e/node_modules/webdriverio/build/middlewares.js:27:57)
[0-0] (node:10140) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:10140) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Can we catch the error or something?

Reply:
fixed the issue and made it so it throws a proper stale element exception